interface/main/calendar/add_edit_event.php

```diff
@@ -1336,7 +1336,15 @@ function find_available(extra) {
         <?php endif ?>
     </ul>
 </nav> <!-- nav-group -->
-<form role="form" method='post' name='theform' id='theform' action='add_edit_event.php?eid=<?php echo attr($eid) ?>'>
+<?php
+$form_id = 'theform';
+if (!empty($_GET['prov']) && ($_GET['prov'] == true)) {
+    $form_id = 'theform_prov';
+} elseif ($_GET['group'] == true) {
+    $form_id = 'theform_groups';
+}
+?>
+<form role="form" method='post' name='<?php echo $form_id; ?>' id='<?php echo $form_id; ?>' action='add_edit_event.php?eid=<?php echo attr($eid) ?>'>
```

Comment:
Change the attr($eid) to be attr_url($eid) to make sure $eid is urlencoded.  I saw that the existing code didn't have it, which looks like an oversight to me if $eid ever changes from a numeric value it will break url encoding.

Where the form_id is defined to close in locality you don't need attrs on those vars but for consistency would be good to have them.